frontend/tests/controllers-test/contactUsCtrl.test.js

```diff
@@ -0,0 +1,193 @@
+        });
+
+        it('when username is invalid', function () {
+            var resetconfirmFormValid = true;
+            tryClauseResponse = usernameInvalid;
+            success = false;
+            vm.user.name = "abc";
+            vm.user.email = "abc@gmail.com";
+            vm.user.message = "Some Other Message";
+
+            vm.contactUs(resetconfirmFormValid);
+            expect(vm.isDisabled).toBeFalsy();
+            expect(vm.isFormError).toBeTruthy();
+            expect(vm.FormError).toEqual(tryClauseResponse.name[0]);
+            expect(vm.stopLoader).toHaveBeenCalled();
+        });
+
+        it('when email is invalid', function () {
+            var resetconfirmFormValid = true;
+            tryClauseResponse = emailInvalid;
+            success = false;
+            vm.user.name = "abc";
+            vm.user.email = "abc@gmail.com";
+            vm.user.message = "Some Other Message";
+
+            vm.contactUs(resetconfirmFormValid);
+            expect(vm.isDisabled).toBeFalsy();
+            expect(vm.isFormError).toBeTruthy();
+            expect(vm.FormError).toEqual(tryClauseResponse.email[0]);
+            expect(vm.stopLoader).toHaveBeenCalled();
+        });
+
+        it('when message is invalid', function () {
+            var resetconfirmFormValid = true;
+            tryClauseResponse = messageInvalid;
+            success = false;
+            vm.user.name = "abc";
+            vm.user.email = "abc@gmail.com";
+            vm.user.message = "Some Other Message";
+
+            vm.contactUs(resetconfirmFormValid);
+            expect(vm.isDisabled).toBeFalsy();
+            expect(vm.isFormError).toBeTruthy();
+            expect(vm.FormError).toEqual(tryClauseResponse.message[0]);
+            expect(vm.stopLoader).toHaveBeenCalled();
+        });
+
+        it('other backend error in try clause', function () {
+            var resetconfirmFormValid = true;
+            tryClauseResponse = {};
+            success = false;
+            vm.user.name = "abc";
+            vm.user.email = "abc@gmail.com";
+            vm.user.message = "Some Other Message";
+
+            vm.contactUs(resetconfirmFormValid);
+            expect(vm.isDisabled).toBeFalsy();
+            expect(vm.isFormError).toBeTruthy();
+            expect($rootScope.notify).toHaveBeenCalledWith("error", "Some error occured. Please try again!");
+            expect(vm.stopLoader).toHaveBeenCalled();
+        });
+
+        it('backend error with catch clause', function () {
+            var resetconfirmFormValid = true;
+            success = null;
+            vm.user.name = "abc";
+            vm.user.email = "abc@gmail.com";
+            vm.user.message = "Some Other Message";
+
+            vm.contactUs(resetconfirmFormValid);
+            expect(vm.isDisabled).toBeFalsy();
+            expect($rootScope.notify).toHaveBeenCalled();
+            expect(vm.stopLoader).toHaveBeenCalled();
+        });
+    });
```

Comment:
Repeated local variables seem to be used like `success` please validated statuses if you intended to do so.